graylog2-web-interface/src/theme/utils/index.ts

```diff
@@ -20,15 +20,13 @@ import type { ColorLevel } from './colorLevel';
 import colorLevel from './colorLevel';
 import type { ContrastingColor } from './contrastingColor';
 import contrastingColor from './contrastingColor';
-import type { Opacify } from './opacify';
 import opacify from './opacify';
 import type { ReadableColor } from './readableColor';
 import readableColor from './readableColor';
 
 export type Utils = {
   colorLevel: ColorLevel,
   contrastingColor: ContrastingColor,
-  opacify: Opacify,
```

Comment:
Are we removing this on purpose? This would break when `GlobalThemeStyles.js` is migrated to TS. Maybe we should fix the typings instead?